designer/client/webpack.config.ts

```diff
@@ -117,19 +117,25 @@ const config: Configuration = {
                 target: "http://localhost:5001",
                 changeOrigin: true,
                 pathRewrite: {
-                    "^/submodules/components": "/",
+                    "^/submodules/components": "",
+                },
+                onError: (err, req, res) => {
```

Comment:
We should redirect in case of any error or only some 404? Maybe at least we should print this error as well to avoid some strange situation where some nasty error is silently hidden?

Reply:
After an offline discussion, @JulianWielga encouraged me, that this situation is already a narrow case and it won't give us too much. He don't see printing errors information as a valuable thing